app/templates/direct-award/view-project.html

```diff
@@ -172,23 +172,10 @@
             ] if project else [])
           },
           {
-            "body": "Complete the Customer Benefits Record form",
+            "body": "Submit a Customer Benefits Record",
             "custom_body_list": [
               {"type": "text", "text":
-                ''.join(["You must fill in the <a href=\"", framework_urls.customer_benefits_record_form_url|e, "\"
```

Comment:
I assume that we want to keep using the url from the frameworks repo rather than hard-coding, so I will create a PR to change that.

Reply:
https://github.com/alphagov/digitalmarketplace-frameworks/pull/595